src/GeositeFramework/js/Launchpad.js

```diff
@@ -30,7 +30,7 @@ require(['use!Geosite'], function (N) {
             this.template = N.app.templates['template-launchpad'];
             this.initialExtent = parseExtent(N.app.data.region.initialExtent);
 
-            if (this.model.get('showByDefault')) {
+            if (this.model.get('showByDefault') && location.hash === '') {
```

Comment:
Checking location.hash is a _little_ fragile because of [this fn](https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/master/src/GeositeFramework/js/App.js#L70) gets called before the LP is initialized, the has would always be set.  Right now the hashModel call is [just the next line](https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/master/src/GeositeFramework/js/App.js#L34) after the LP init.  Might be a little more future proof to check for hash contents first thing in App and save that bool as an attr on the LP model.